source/wave_energy.rst

```diff
@@ -148,19 +146,21 @@ First we describe required inputs.  The required inputs are the minimum data nee
 	File type: *.csv
 	Sample data set:  \InVEST\WaveEnergy\Input\Machine_Pelamis_Performance
 
-.. figure:: ./wave_energy/table_pelamisperformance.png
-   :align: center
-   :figwidth: 500px
+.. csv-table::
```

Comment:
The indentation between the numbered inputs and the associated tables doesn't look great, is it possible to bump out the tables too? I also wonder if using your modifying approach might be good here to truncate some of these larger tables where more rows / cols doesn't really provide a lot of benefit?

Reply:
I had tried to line up the tables within the numbered items and had no luck but someone on stackoverflow just pointed me in the right direction so it's fixed now!